packages/cli/src/util/dev/server.ts

```diff
@@ -1280,9 +1280,20 @@ export default class DevServer {
     const method = req.method || 'GET';
     this.output.debug(`${chalk.bold(method)} ${req.url}`);
 
+    // `print` without a newline so that we can finish the line later with an error or a status code
+    this.output.print(`${method} ${chalk.bold(req.url)} :: `);
+
     try {
       const vercelConfig = await this.getVercelConfig();
       await this.serveProjectAsNowV2(req, res, requestId, vercelConfig);
+
+      // wait until the response completes so that the `statusCode` has the real response value
+      res.on('close', () => {
```

Comment:
Should this listener be added before the `try` instead?

Reply:
I don't want the callback to be executed unless we've called `serveProjectAsNowV2` because it will think the `res.statusCode` is `200`, which would be a very false positive. I had some issues with this during development.